src/common/evaluation/QA/eval_depthmap_models/src/qa_config_height.py

```diff
@@ -1,43 +1,53 @@
 from bunch import Bunch
 
-#Details of model used for evaluation
+# Details of model used for evaluation
 MODEL_CONFIG = Bunch(dict(
     EXPERIMENT_NAME='q3-depthmap-plaincnn-height-95k',
 
     RUN_ID='q3-depthmap-plaincnn-height-95k_1597988908_42c4ef33',  # Run 3
-    #RUN_ID = 'q3-depthmap-plaincnn-height-95k_1600451633_cb44f6db',     #Run 17
+    # RUN_ID = 'q3-depthmap-plaincnn-height-95k_1600451633_cb44f6db',     #Run 17
 
     INPUT_LOCATION='outputs',
     NAME='best_model.h5',
 ))
 
+FILTER_CONFIG = Bunch(dict(
+    SWITCH='ON',  # 'OFF'
```

Comment:
We can use a binary variable instead. 
```suggestion
    IS_ENABLED=True,
```